src/data/actions/auth.js

```diff
@@ -88,11 +89,15 @@ export const currentUser = (client) => (dispatch) => {
   return AuthService.getCurrentUser(client).then(
     (res) => {
       //login successful
-      if (res.data.getActiveUser !== null){ 
+      if (res.data.getActiveUser !== null){
         dispatch({
           type: LOGIN_SUCCESS,
           payload: { 
+            firstName: res.data.getActiveUser.firstName,
+            lastName: res.data.getActiveUser.lastName,
             email: res.data.getActiveUser.email,
+            userId: res.data.getActiveUser.id,
+            teamId: res.data.getActiveUser.teamId,
```

Comment:
nit: might be easier for maintainability if we destructure the object for the values we need and then assign them accordingly
```suggestion
      if (res.data.getActiveUser !== null){
        const { id, firstName, lastName, email, teamId} = res.data.getActiveUser;
        dispatch({
          type: LOGIN_SUCCESS,
          payload: { 
            firstName,
            lastName,
            email,
            userId: id,
            teamId,
```

Reply:
This is a lot cleaner, fixed! Thanks